src/stories/Editor.stories.tsx

```diff
@@ -11,53 +13,130 @@ const content = `
   <strong><span style="font-size: 14pt;"><span style="color: #7e8c8d; font-weight: 600;">No matter what you're building, TinyMCE has got you covered.</span></span></strong>
 </p>`;
 
+let lastChannel = '5';
+const getConf = (stringConf) => {
+  let conf  = {};
+  console.log('parsing: ', stringConf);
+  try {
+    conf = JSON.parse(stringConf);
```

Comment:
Will JSON be enough? That won't work for functions.

Reply:
Do we to have functions for testing purposes only? We can always change it to eval instead

Reply:
Move to use `new Function()` instead